src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -641,13 +498,13 @@ export default {
 		async handleSubmitSpam(numberOfMessages) {
 			console.debug('Sending ' + numberOfMessages + ' lorem ipsum messages')
 			for (let i = 0; i < numberOfMessages; i++) {
-				const randomNumber = parseInt(Math.random() * 500, 10)
+				const randomNumber = parseInt((Math.random() * 500).toString(), 10)
```

Comment:
```suggestion
				const randomNumber = Math.floor(Math.random() * 500)
```

Or tricky but short 😀

```suggestion
				const randomNumber = ~~(Math.random() * 500)
```